packages/js/admin-layout/src/gutenberg/components/index.ts

```diff
@@ -0,0 +1,14 @@
+export * from './section-tabs';
+export * from './header';
+
+/**
+ * External dependencies
+ */
+/* eslint-disable @woocommerce/dependency-group */
+// @ts-expect-error missing type.
+import SidebarNavigationScreen from '@wordpress/edit-site/build-module/components/sidebar-navigation-screen';
+// @ts-ignore No types for this exist yet.
+import SidebarNavigationItem from '@wordpress/edit-site/build-module/components/sidebar-navigation-item';
+/* eslint-enable @woocommerce/dependency-group */
+
+export { SidebarNavigationScreen, SidebarNavigationItem };
```

Comment:
Here you are importing two components from `edit-site` and bundling them together with a Woo script. There's not really a runtime dependency on `wp.editSite` and the `wp-edit-site` script. If the components are interesting for something that is not Site Editor at all, they should probably live somewhere else. Some private/experimental `@wordpress/sidebar` package?

Other components from `edit-site` that are used here are `SidebarContent` and `SiteHub`. Also very generic things that are related to "admin sidebar" rather than a specialized admin screen.